.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-* @lolgab @ljmf00 @andreaTP @rtfpessoa @bmbferreira @DReigada @pedrocodacy
+* @lolgab @rtfpessoa @pedrocodacy
 
 *.yml @h314to @paulopontesm
 
```

Comment:
We use the @codacy/toss team as code owner in the other repositories:
```suggestion
* @codacy/toss
```
This way is resistant to offboardings :)

Reply:
Love it <3